cypress/integration/pages/liveRadio/testsForCanonicalOnly.js

```diff
@@ -60,6 +61,7 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
 
         if (isRadioScheduleOnPage) {
           const schedulePath = Cypress.env('currentPath')
+            .concat(`${dataEndpointOverride()}`)
```

Comment:
You can remove the ${}
```suggestion
            .concat(dataEndpointOverride())
```